incubator/fluentd/values.yaml

```diff
@@ -87,7 +88,7 @@ configMaps:
       logstash_format true
       <buffer>
         @type file
-        path /var/log/fluentd-buffers/kubernetes.system.buffer
+        path "#{ENV['OUTPUT_BUFFER_PATH']}/kubernetes.system.buffer"
```

Comment:
If we can mount the PV where ever we want why convert this to a configurable parameter?

Reply:
I'll keep it hardcoded, my idea was that people can use that variable on the config files as base path for buffers. Just to make it handy